Jenkinsfile

```diff
@@ -64,6 +64,9 @@ sudo chown core:core -R /pathHadoop/
 QUICK_START_PATH=${JENKINS_HOME}/${BED}/singlebox/quick-start
 CONFIG_PATH=${JENKINS_HOME}/${BED}/singlebox/cluster-configuration
 
+# Install necessary python package
+sudo pip install GitPython
```

Comment:
Could we remove the line? Noticed that you have updated the dependencies in `dev-box`.

Reply:
No we can't. paictl need import it. You will call paictl to generate cluster-configuration out of the dev-box.